client/js/components/toggle-show-hide.js

```diff
@@ -0,0 +1,22 @@
+import showHide from './show-hide';
+import { trackEvent } from '../utils/track-event';
+
+export default (el) => {
+  const toggle = showHide({el});
+  const trackingJson = el.getAttribute('data-tracking-info');
```

Comment:
`tracking-info` was something I created here as separate from the generic `js-track-click` stuff. How would I otherwise get the e.g. `componentId` and `section` heading from the drawer component?